src/gsi/read_satwnd.f90

```diff
@@ -1242,6 +1295,7 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
 
            ! assign types and get quality info : end
 
+99         continue
```

Comment:
A comment providing a callback to where the code is routing to this line from would be helpful to avoid confusion: "From PCCF check on Line XXX" or similar.

Can you also give a brief explanation of why this routing to 99 is being implemented based on a check for PCCF? Is this a necessary change for the satwnd data being ingested with this PR, or is it for something else?

Reply:
@BrettHoover-NOAA  Its about backward compatibility. When assimilating satwnds from before 2010 you can run into problems with older satwnd definitions which had no PCCF defined getting thrown away. My solution was if PCCF is not defined, skip the part of the code that processes that information . I could put a comment in about it.

Reply:
All true. Checking PCCF because it catches other satellites missing QC information, such as  meteosat, mtsat, gms, as well as goes.

Reply:
Hi @jack-woollen I appreciate the explanation! Is it possible to reduce the scope of the `goto` command to be more specific to assimilating older winds? My only outstanding concern would be that this re-routing would also apply to current GOES series winds (NC00503*) that *should* have a PCCF value but for whatever reason do not, and should be removed in QC rather than passed through with the `goto` exception.

Reply:
hi @BrettHoover-NOAA Likewise. I took a look at PCCF distribution in a 00z satwnd file and there are a small number of GOES16 and GOES18 winds which have missing PCCF, and all the reports from NPP and N20 have no PCCF.  So you're saying you toss the winds with missing PCCF because they might be bad. What about if you want to use them?

I can put a time window on letting winds with missing PCCF through, like only before 2012 or thereabouts. I agree, that's the easiest way to fix this right now.

Reply:
It could also key on certain SAIDs, as mentioned by @ilianagenkova .

Reply:
@jack-woollen @jswhit There is a standing prohibition on assimilating GOES or polar winds that are missing PCCF values if it is expected that they should be there, because the percent confidence is a necessary QC filter. Often bad winds are accidentally packed into the BUFR file with an unassigned PCCF value, possibly because they didn't get far enough into QC on the production side to be assigned one, and those need to be removed.

We can probably approach this using satellite ID number (SAID) and the BUFR tank for both GOES and polar winds to narrow the scope:

GOES: AMVs from both old and new products are in NC00503* tanks, but old products are identified by SAID<270. So AMVs from these tanks with SAID>=270 are expected to have PCCF values and should be QC'd normally while those with SAID<270 can pass through with the `goto` exception.

Polar: AMVs from old products come from tanks NC0057*, NC00580, NC00590 - these are not expected to have a PCCF value and can pass through with the `goto` exception. Other polar AMVs are expected to have a PCCF value and should be QC'd normally.

Reply:
@BrettHoover-NOAA Below is the GDAS satwnd dump data for 00z. I'll apply the rules above to the tanks.
NC005044        1022         49923       9990884       48.85
NC005045         614         30010       6005758       48.88
NC005046        1269         62104      12428488       48.94
NC005067        4770        128671      47351166       26.98
NC005068         528         14143       5204850       26.79
NC005069        8218        221825      81631722       26.99
NC005070          55          3754        521022       68.25
NC005071          14           952        132132       68.00
NC005080          41          2542        353010       62.00
NC005081         300         62880       2941824      209.60
NC005091         396         87323       3883378      220.51
NC005024        1949         95349      19081644       48.92
NC005025           2            23          4686       11.50
NC005026        1513         74043      14817770       48.94
NC005030        2271        518175      22621502      228.17
NC005031         587        153436       5811322      261.39
NC005032        6645       1837290      66243336      276.49
NC005034         897        214272       8919966      238.88
NC005039         305         75307       3013054      246.91
NC005072         268         72319       2596188      269.85